README.md

```diff
@@ -1,17 +1,119 @@
-# Griptape Tool Template
+# Griptape Extension Template
 
-A Github template repository for creating Griptape tools.
+A Github template repository for creating Griptape extensions.
 
-## Usage
+## Getting Started
 
-This template repository defines the typical structure for a Griptape tool and is meant to be used as a starting point when developing new tools. To create a tool using this template, click the **"Use this template"** button at the top of the repository.
+Via github web page:
 
-### Steps for creating a new Griptape tool:
+Click on `Use this template`
+
+![](https://docs.github.com/assets/cb-36544/images/help/repository/use-this-template-button.png)
+
+
+Via `gh`:
+
+```
+$ gh repo create griptape-extension-name -p griptape/griptape-extension-template
+```
+
+## What is a Griptape Extension?
+
+Griptape Extensions can add new functionality to the [griptape framework](https://github.com/griptape-ai/griptape), such as new Tools, Drivers, Tasks, or Structures.
```

Comment:
Be clear what that affords. "With Griptape Extensions, your Agents can now do X, Y, Z."

Also capitalize `Griptape framework`

Reply:
What do you have in mind for X, Y, Z? Extensions don't do anything concrete on their own, they simply enable the user to develop functionality like new Tools, Drivers, Tasks, or Structures.

Reply:
right, but that may not be obvious what that affords. "With extensions, your Griptape Agents can leverage your team's APIs, perform custom business logic, and access proprietary data sources, all as part of their reasoning."

Reply:
Added a sentence.